### PR TITLE
Code coverage for RustPython

### PR DESCRIPTION
**Pull request by calixteman**
Attempt to bring code coverage

### CODE REVIEW

**Comment by calixteman** [comment]
Need to fix a bug in grcov to avoid file names conflicts:
https://github.com/mozilla/grcov/pull/254

**Comment by calixteman** [comment]
For any reason, the ccov doesn't seem to be correct:
https://codecov.io/gh/RustPython/RustPython/src/31c5b5ed3d1716fc9575ab480ae9a41ca838d421/vm/src/obj/objlist.rs
So need to investigate to understand what's wrong here.

**Comment by calixteman** [comment]
Seems to work now:
https://codecov.io/gh/RustPython/RustPython/src/e86e108e67b016b3befa4654d7fe606f5e04c733/vm/src/obj/objlist.rs

File: README.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 A Python-3 (CPython >= 3.5.0) Interpreter written in Rust :snake: :scream: :metal:.
 
 [![Build Status](https://travis-ci.org/RustPython/RustPython.svg?branch=master)](https://travis-ci.org/RustPython/RustPython)
+[![codecov](https://codecov.io/gh/RustPython/RustPython/branch/master/graph/badge.svg)](https://codecov.io/gh/mozilla/RustPython/RustPython)
```
Comment (adrian17) [inline README.md]: `mozilla/RustPython/RustPython`?
Seems like the link is dead.

Comment (calixteman) [reply README.md]: my bad, bad copy/paste...

Comment (calixteman) [reply README.md]: I just made a PR to fix that:
https://github.com/RustPython/RustPython/pull/491

**Pull request merged**